mobile/src/main/java/org/openhab/habdroid/ui/AbstractBaseActivity.kt

```diff
@@ -75,11 +74,7 @@ abstract class AbstractBaseActivity : AppCompatActivity(), CoroutineScope {
 
     @CallSuper
     override fun onCreate(savedInstanceState: Bundle?) {
-        setTheme(getActivityThemeId())
-        if (getPrefs().getBoolean(PrefKeys.DYNAMIC_COLORS, false)) {
-            DynamicColors.applyToActivityIfAvailable(this)
-        }
-
+        applyUserSelectedTheme()
```

Comment:
Should additionally apply dynamic colors in `onStart` (palette can change at runtime)

Reply:
Further research suggests this isn't actually needed (e.g. `DynamicColors.applyToActivitiesIfAvailable` also does this only in `onPreCreate`). I've seen an old color scheme being stuck one or two times, though; will monitor this.